src/Ng/Arguments.cs

```diff
@@ -1,64 +1,102 @@
 ﻿namespace Ng
 {
-    static class Constants
+    static class Arguments
     {
+
+        #region Shared
```

Comment:
I sorted everything by what job they were in, using regions, which is much prettier than the alphanumerically sorted blob of arguments before. If you think of a better way to do this let me know.